fpdf/text_region.py

```diff
@@ -156,8 +156,9 @@ def generate_bullet_frags_and_tl(self, bullet_string: str, bullet_r_margin: floa
             bullet_fragments,
             max_width=self._region.get_width,
             margins=(
-                self.pdf.c_margin + (self.indent - fragments_width - bullet_r_margin),
-                self.pdf.c_margin,
+                self._region.h_margins[0]
+                + (self.indent - fragments_width - bullet_r_margin),
```

Comment:
Shouldn't `indent=self.indent` be provided there, instead of adding `self.indent` to the left margin?